test/bin/mirror_registry.sh

```diff
@@ -79,10 +85,22 @@ EOF
 mirror_images() {
     local -r ifile=$1
     local -r ofile=$(mktemp /tmp/container-list.XXXXXXXX)
+    local -r ffile=$(mktemp /tmp/from-list.XXXXXXXX)
+
+    # Add the distribution registry and non-localhost-FROM images
+    # to the mirrored list explicitly
+    echo "${REGISTRY_IMAGE}" > "${ffile}"
```

Comment:
This image is already local to the hypervisor, right? https://github.com/openshift/microshift/pull/4225/files#diff-25478684142fbd29b423928d694845c40d6092a0fda08b781e0a70a770808c30R26
Do we need it in the registry itself? Its the running image for that.

Reply:
Yes, it makes sense only if we're caching images in S3.
Since we decided not to do the S3 caching, I will remove it.